uint64 id_0
sensor_msgs/Image left_0
sensor_msgs/CameraInfo info_left_0
sensor_msgs/Image right_0
sensor_msgs/CameraInfo info_right_0

uint64 id_1
sensor_msgs/Image left_1
sensor_msgs/CameraInfo info_left_1
sensor_msgs/Image right_1
sensor_msgs/CameraInfo info_right_1

